log-engine-tuulbox/build.gradle.kts

```diff
@@ -0,0 +1,63 @@
+plugins {
+    // Android plugin must be before multiplatform plugin until https://youtrack.jetbrains.com/issue/KT-34038 is fixed.
+    id("com.android.library")
+    kotlin("multiplatform")
+    id("kotlinx-atomicfu")
+    id("org.jmailen.kotlinter")
+    id("org.jetbrains.dokka")
+    id("com.vanniktech.maven.publish")
+}
+
+kotlin {
+    explicitApi()
+
+    android {
+        publishAllLibraryVariants()
+    }
+    js().browser()
+    iosX64()
+    iosArm32()
+    iosArm64()
+    macosX64()
+
+    sourceSets {
+        val commonMain by getting {
+            dependencies {
+                api(project(":core"))
+                implementation(tuulbox("logging"))
+            }
+        }
+
+        all {
+            languageSettings.enableLanguageFeature("InlineClasses")
+        }
+    }
+}
+
+atomicfu {
+    transformJvm = true
+    transformJs = false
+}
+
+android {
+    compileSdkVersion(30)
+
+    defaultConfig {
+        minSdkVersion(21)
+    }
+
+    buildFeatures {
+        buildConfig = false
+    }
```

Comment:
nit: I believe this can be configured via `gradle.properties`? If so, would be better to do it there and drop this configuration in the other modules as well.